styles/abstracts/_theme.scss

```diff
@@ -37,6 +37,7 @@ $themes: (
     'blue-accent-bg-color': #b6cbff,
     'blue-light-hover-color': #e8edfb,
     'card-hover-opacity': 0.85,
+    'separator-line-color': #cccccc,
```

Comment:
can't reuse an existing color?

Reply:
#cccccc in white mode and #6b6b6b in dark mode. There is no existing colour to be used.